.travis.yml

```diff
@@ -45,12 +45,14 @@ jobs:
         - sudo apt-get -y install gperf swig
         - if [[ ! -e "./iverilog/README.txt" ]]; then rm -rf iverilog; git clone https://github.com/steveicarus/iverilog.git --depth=1 --branch v10_3; fi
         - cd iverilog && autoconf && ./configure && make -j2 && sudo make install && cd ..
-        - pip install tox
+        - pip install tox codecov virtualenv --upgrade
```

Comment:
Nit: Can you put the `--upgrade` flag right after `pip install`?

Reply:
Done